tests/containers/podman_netavark.pm

```diff
@@ -0,0 +1,143 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman, netavark, aardvark
+# Summary: Test podman netavark network backend
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp is_transactional);
+use containers::utils qw(get_podman_version);
+use transactional qw(trup_call check_reboot_changes);
+use utils qw(zypper_call);
+use Utils::Systemd qw(systemctl);
+
+sub remove_subtest_setup {
+    assert_script_run("podman container rm -af");
+    assert_script_run("podman network prune -f");
+    assert_script_run("podman network ls");
+    assert_script_run("podman ps -a");
+}
+
+sub _cleanup {
+    my $podman = shift->containers_factory('podman');
+    select_console 'log-console';
+    remove_subtest_setup;
+    script_run('rm -rf /etc/containers/containers.conf');
+    $podman->cleanup_system_host();
+    validate_script_output('podman info --format {{.Host.NetworkBackend}}', sub { /cni/ });
+    validate_script_output('podman network ls', sub { /podman\s+bridge/ });
+}
+
+
+sub run {
+    my ($self, $args) = @_;
+
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+    my $podman_version = get_podman_version();
+
+    if (package_version_cmp($podman_version, '4.4.2') < 0) {
+        record_info('No support', "Netavark backend is not supported in podman-$podman_version");
+        return 1;
+    }
+
+    my @pkgs = qw(netavark aardvark-dns);
+
+    if (is_transactional) {
+        trup_call("pkg install @pkgs");
+        check_reboot_changes;
+    } else {
+        zypper_call("in @pkgs");
+    }
+
+    $podman->cleanup_system_host();
+
+    ## change network backend to *netavark*
+    assert_script_run(q(echo -e '[Network]\nnetwork_backend="netavark"' > /etc/containers/containers.conf));
+    validate_script_output('podman info --format {{.Host.NetworkBackend}}', sub { /netavark/ });
+
+    ## TEST1
+    ## set static IP, and MAC addresses on a per-network basis
+    my $testnet = 'test_simple';
+    my $test_ctr = 'docker.io/fedora/apache';
```

Comment:
```suggestion
    my $test_ctr = registry_url('fedora/apache');
```
Our `REGISTRY` should pull the image from `docker.io` but cache it.

Reply:
Does it make sense as the image is pulled only once?

Reply:
Yes, because we can't tell if there are other tests pulling images from docker.io, it could be from tests or even from some other places. We can't rely on docker.io any more.

Reply:
ok, so the image got injected in our registry without `library`, so we can't use `registry_url` ... so, it's just `fedora/apache`..